src/format.rs

```diff
@@ -207,19 +207,21 @@ impl<'a> Formatter<'a> {
                     nil()
                 } else {
                     ".{".to_doc()
-                        .append(concat(
-                            unqualified
-                                .iter()
-                                .map(|e| e.clone().to_doc())
-                                .intersperse(", ".to_doc()),
-                        ))
+                        .append(
+                            concat(
+                                unqualified
+                                    .iter()
+                                    .map(|e| {
+                                        break_(&e.clone().name, &e.clone().name)
+                                            .append(import_as(&e.clone().as_name))
```

Comment:
We can reduce the amount of memory allocation like so:
```suggestion
                                        e.name
                                            .to_doc()
                                            .append(break_("", ""))
                                            .append(import_as(&e.as_name))
```